release/nightly_tests/dataset/map_batches_benchmark.py

```diff
@@ -0,0 +1,149 @@
+from typing import Literal, Optional, Union
+
+import ray
+from ray.data._internal.compute import ActorPoolStrategy, ComputeStrategy
+from ray.data.dataset import Dataset
+
+from benchmark import Benchmark
+
+
+def map_batches(
+    input_ds: Dataset,
+    batch_size: int,
+    batch_format: Literal["default", "pandas", "pyarrow", "numpy"],
+    compute: Optional[Union[str, ComputeStrategy]] = None,
+    num_calls: Optional[int] = 1,
+) -> Dataset:
+
+    ds = input_ds
+    for _ in range(num_calls):
+        ds = ds.map_batches(
+            lambda ds: ds,
```

Comment:
Can we name it to 'batch' or just 'x'? Looks too many 'ds' around.

Reply:
updated.